common/script/index.coffee

```diff
@@ -1692,8 +1809,16 @@ api.wrap = (user, main=true) ->
       # Analytics
       user.flags.cronCount?=0
       user.flags.cronCount++
-      options.mixpanel?.track('Cron',{'distinct_id':user._id,'resting':user.preferences.sleep})
-      options.ga?.event('behavior', 'cron', 'cron', user.flags.cronCount).send(); #TODO userId for cohort
+
+      analyticsData = {
+        category: 'behavior',
+        gaLabel: user.flags.cronCount,
```

Comment:
A GA label is usually a string (sort of a subcategory); they have a "value" field for numeric subdivisions, if we'd like to track this there.